### PR TITLE
Added components for pages to welcome parents/instructors and continue registration process

### PR DESCRIPTION
**Pull request by am-stewart**
## Description

This pull request introduces two new components to the registration flow. I have added two welcome pages, one for the instructors and one for the parents, that they will see after verifying their email from the Okta link they received. The pages will each have a progress bar at the top that will indicate to the user where they are in the registration process and how much further they have but these have not been added yet. Each page also has a button that will take them to the next step in registration. Currently only one is functioning since the next step in the instructor registration flow has not been added yet.

## Loom Video

https://www.loom.com/share/3fdd59f7aa0e4c42bf0b6359d20701a6

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/Registration/InstructorWelcome.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+export default function InstructorWelcome() {
+  return (
+    <div className="reg-content-container">
+      <div className="content">
+        <p className="color-one">Welcome to CoderHeroes!</p>
+        <p className="color-two">
+          You are steps away from sharing your superpowers with future
+          CoderHeroes
+        </p>
+        <a href="#">CREATE ACCOUNT</a>
```
Comment (Timbobeek) [inline src/components/pages/Registration/InstructorWelcome.js:13]: This probably needs a comment to make it easier to replace this placeholder later.

File: src/index.js [hunk 2]
```diff
@@ -90,6 +92,8 @@ function App() {
           <Route exact path="/" component={LandingPage} />
           <Route path="/login" component={LoginPage} />
           <Route path="/register-2" component={HowManyStudents} />
+          <Route path="/instructor-register-1" component={InstructorWelcome} />
```
Comment (Timbobeek) [inline src/index.js:95]: Perhaps we all need to make the index.js routes more organized, put in chronological order or group them up based on the parts of the website they are responsible for... guilty myself!

**Review by Timbobeek — APPROVED** [review]
Overall, clean code, seems on point to me. I left a few comments.

Once this PR gets approved, remember to add the progress bar designed by David I believe, as well as the redirect to that button you have on your page (one of the hrefs I commented on). Feel free to ask any questions!

**Review by ElchinNasirov — APPROVED** [review]

File: src/index.js (hunk 2 above)
Comment (luisbryandr) [reply src/index.js:95]: This is a good point, would only help our fellow developers read and understand more smoothly.

File: src/components/pages/Registration/InstructorWelcome.js (hunk 1 above)
Comment (luisbryandr) [inline src/components/pages/Registration/InstructorWelcome.js:13]: Not sure if I am the only one seeing it but I get a linting warning from Github, might be worth visiting before merging this PR.

Comment (luisbryandr) [reply src/components/pages/Registration/InstructorWelcome.js:13]: I see there are multiple now, and prob a recurring and okay theme throughout these merges/ PRs

**Review by luisbryandr — APPROVED** [review]
I don't see anything wrong or more to add aside from Tim's comments. Thanks for your work!

File: src/components/pages/Registration/ParentWelcome.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import RegistrationProgress from './RegistrationProgress';
```
Comment (kfam22) [inline src/components/pages/Registration/ParentWelcome.js:2]: I just added the registration progress component

File: src/index.js [hunk 4]
```diff
@@ -90,6 +92,8 @@ function App() {
...
+          <Route path="/register-1" component={ParentWelcome} />
```
Comment (kfam22) [inline src/index.js:96]: route is consistent with the rest of the registration flow, great job!

File: src/components/pages/Registration/ParentWelcome.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+import '../../../styles/registration.less';
+
+export default function ParentWelcome() {
+  return (
+    <div className="reg-content-container">
```
Comment (kfam22) [inline src/components/pages/Registration/ParentWelcome.js:7]: Formatting is consistent with the rest of the registration flow!

**Review by kfam22 — APPROVED** [review]
Great job adding improvements and keeping this page consistent with the rest of the registration flow.

**Pull request merged**